src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -78,6 +78,15 @@ public Version getVersion(String version)
 
     static class WriterFactory extends SSTableWriter.Factory
     {
+        @Override
+        public long estimateSize(SSTableWriter.SSTableSizeParameters parameters)
+        {
+            return (long) ((parameters.partitionCount() // index entries
+                            + parameters.partitionCount() // keys in data file
```

Comment:
Shouldn't this be `parameters.partitionKeySize()`, which isn't called anywhere? Also, I would call that method `partitionKeysSize`, in plural, so one can't think that this somehow refers to the size of some single partition.